Project1/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<body>
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link href="https://fonts.googleapis.com/css?family=Bitter" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Bitter|Bree+Serif|EB+Garamond:600|PT+Serif" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Bitter|Bree+Serif|PT+Serif" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Bitter|PT+Serif" rel="stylesheet">
+        <link rel="stylesheet" href="styles.css">
+        <title>Document</title>
+    </head>
+
+    <figure>
+        <img src="images/maldivesoctopus.jpg" alt="an octopus swims in the ocean.">
+        <figcaption>This is an octopus.</figcaption>
+    </figure>
+
+    <h1>
+        <strong>Yes, the Octopus Is Smart as Heck. But Why?</strong>
```

Comment:
`strong` should be reserved for items that have urgency or seriousness; the `h1` tag already conveys the importance of this text, so no further markup is needed:
```suggestion
        Yes, the Octopus Is Smart as Heck. But Why?
```
Read more about [`strong` here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong)